frontend/src/components/Tool/Tools.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import {
+    Card, CardBody, CardTitle, Row, Col
+  } from 'reactstrap'
+    
+import '../../style/Tools.css'
+import reactLogo from './Tool_logos/React_logo.jpg'
+import nodeLogo from './Tool_logos/NodeJS_logo.png'
+import jiraLogo from './Tool_logos/Jira_logo.jpg'
+import postmanLogo from './Tool_logos/Postman_logo.jpg'
+import awsLogo from './Tool_logos/AWS_logo.jpg'
+import herokuLogo from './Tool_logos/Heroku_logo.jpg'
+
+function Tools() {
+    
+    return (
+        <div>
+            <h2 className="maintitle">Tools Used:</h2>
+                <Row>
+                <Col sm="1" className="centercard">
```

Comment:
I see that you used sm="1". What made you go with that instead of sm="2" or sm="4"